lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service.rb

```diff
@@ -201,11 +201,11 @@ def datastore
     def ldap
     end
 
-    def execute(method_name, *args)
+    ruby2_keywords def execute(method_name, *args)
```

Comment:
TODO:  try removing ruby2_keywords and see if I can change options hash methods in `MiqAeServiceMethods` to use kwargs and use `**` in caller delegates such as this.

```
(method_name, *args, **kwargs)
```

Reply:
I've dropped ruby2_keywords and found a temporary solution I'm fine with.  We basically accept either kwargs/options hashes coming into the execute methods but call the underlying method with only args appended with a hash of kwargs if kwargs exist.

I had to change one method to do no longer accept kwargs and instead pass an options hash.